youtube_dl/extractor/indavideo.py

```diff
@@ -9,8 +11,14 @@
 )
 
 
-class IndavideoEmbedIE(InfoExtractor):
```

Comment:
I understand it. Due backward compability I added back the IndavidedoEmbedIE class.